editors/code/src/bootstrap.ts

```diff
@@ -205,3 +291,8 @@ async function patchelf(dest: vscode.Uri): Promise<void> {
         },
     );
 }
+
+export const _private = {
+    earliestToolchainPath,
+    orderFromPath,
+};
```

Comment:
This appears to be the TS way of "hiding" things that need to be exported purely for testing.